xml/docu_styleguide.structure.xml

```diff
@@ -790,6 +790,43 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
 
  <sect2 xml:id="sec-link">
   <title>External links</title>
+  <para>
+   Information that is relevant within &suse; documentation is often
+   available from other Web sites already.
+   In such cases, choose between linking to these sites or including their
+   content in edited form.
+   Adhere to the following guidelines when selecting sites to link to:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Link to credible sources, such as suse.com, upstream projects, or
+     developer sites.
+     Where possible, avoid linking to direct competitors of &suse;.
+     Do not link to obvious clickbait sites.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Prefer larger, well-kept sites over blogs that may vanish overnight.
+     If you need to link to smaller blogs, save an archive version of the
+     site at <link xlink:href="https://web.archive.org/"/>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Where possible, avoid linking to sites that feature controversial or
+     political content.
+     Make sure to also check the site with your ad-blocker disabled, to see
```

Comment:
I suggest to remove that sentence. Site owners often cannot control what ads get displayed.

Reply:
To clarify: The ads usually depend on the client visiting the page, not on the server hosting it.

Reply:
Sentence removed -> resolved.